### ISSUE TITLE
Tried using the website, always stuck on loading.

### ISSUE THREAD

**Issue by issatillie** [vantezzen/wrapped#2]
I exported my data, but for some reason after exporting it ALWAYS keeps saying 

We're preparing your Wrapped...

Uh oh, this is taking longer than expected...

This should've been done by now...
If it doesn't start soon try reloading
the page and uploading your TikTok data export again.

This is really disappointing... I really wanted to see my activity on the platform, guess my ZIP file is too large for it?
I tried it on my phone, laptop, and in an incognito browser, but nothing seems to work.
On my phone, it says 'Something doesn't look right' when it should be uploaded in a ZIP folder, right? I did just that..

**Comment by vantezzen** [comment]
Thank you for reporting this, can you give me this information if possible:
- When the loading screen is stuck, can you open the JavaScript console and screenshot/copy the error (if any) you are getting there?
- How large is your ZIP file?
- What OS + browsers are you using?

Hopefully this will help me in finding the problem you are facing

**Comment by issatillie** [comment]
Oh hi!

- It now says 'something isn't right, like what I had on my phone before... I downloaded the new ZIP file, it says the same message. Really weird. It also does it with the old ZIP, while before, I was stuck on a loading screen.

  - I Found some console code:
  - SyntaxError: Unexpected token 'D', "Date: 2023"... is not valid JSON
    at JSON.parse (<anonymous>)
    at A.fromZip (page-c0c5c922c2856651.js:1:12359)
    at async onFileSelect (page-c0c5c922c2856651.js:1:22960)
 I think this might be the issue. Line it is referring to:
t && t.apply(d.console, r) 'undefined'


- My ZIP file is 1.84 MB.
- I'm using Windows 10 and Chrome. I tested it on Microsoft Edge, same issue.
